### PR TITLE
Tweak owners

### PR DESCRIPTION
**Pull request by tengqm**
This PR tweaks the OWNERS files by

- removing names that are no longer active in this or the website community.
- adding all SIG Docs leads to the approvers list

(edited)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 # auto-merge happen if a /lgtm exists, or vice versa, or they can do both
 # No need for approvers to also be listed as reviewers
 approvers:
-- bradamant3
-- bradtopol
+- jimangel
 - kbarnard10
-- steveperry-53
+# - kbhawkey
+# - onlydole
```
Comment (kbhawkey) [inline OWNERS:14]: @tengqm , Do you want to remove the pound signs?

Comment (tengqm) [reply OWNERS:14]: 'kbhawkey', 'onlydole' and 'sftim' are not members of 'kubernetes-sigs', so I was getting "do-not-merge/invalid-owners-file" label for this PR.

Comment (kbhawkey) [reply OWNERS:14]: OK. Do you know how we become "members"?

Comment (tengqm) [reply OWNERS:14]: I've just dug into the issue. It looks like `https://github.com/kubernetes/org/blob/master/config/kubernetes-sigs/org.yaml#L632-L644` is the place to edit it. Will sync with @zacharysarah on the next step.

Comment (zacharysarah) [reply OWNERS:14]: @tengqm Thanks for raising visibility--you are correct, the file in k/org sets admin/write permissions for this repo.

@kbhawkey @onlydole @sftim If I understand correctly, you need to:

1. Open an issue in k/org to [apply for membership](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) in the kubernetes-sigs org. Feel free to assign me and @tengqm for a +1.

2. Once you're approved as members of kubernetes-sigs, you should be add-able to `OWNERS` and `org.yaml`. 

Do you need write permissions to generate reference docs?

@tengqm Thanks for tracking down the issue. 🙇

Comment (kbhawkey) [reply OWNERS:14]: No, not to generate the docs.

Comment (onlydole) [reply OWNERS:14]: GitHub issue filed at: https://github.com/kubernetes/org/issues/2006

**Comment by zacharysarah** [comment]
@nikhita 👋 Can you shed light on the [membership situation](https://github.com/kubernetes-sigs/reference-docs/pull/162#issuecomment-655904503)?

I thought https://github.com/kubernetes/org/pull/2002/files#diff-fec2bf18d37c4aa0eba66d7ebe28f3c6R411 added @sftim as a member of k/kubernetes-sigs. Did I miss something?

**Comment by nikhita** [comment]
@zacharysarah -- @sftim would need to accept the invite to join k-sigs first. Once they accept the invite and become a member, running `/verify-owners` will remove the `do-not-merge/invalid-owners-file` label.

**Comment by sftim** [comment]
Invitation received and accepted.

/verify-owners

**Comment by tengqm** [comment]
Ha, forgot that the PR was authored by me. :(

**Comment by nikhita** [comment]
Based on previous PRs and discussion:

/lgtm

**Comment by nikhita** [comment]
Dummy comment to check if triggers tide :thinking:

**Pull request merged**